logging-native/src/commonMain/kotlin/mu/KLogger.kt

```diff
@@ -0,0 +1,130 @@
+package mu
+
+public expect interface KLogger {
```

Comment:
Create documentation:
- logger infrastructure description
- logger available functions usage brief explanation (simple log, with marker, throwable, exit)
- log levels & log modules configuration

**EDIT**:  I noticed description in `logging/src/commonMain/kotlin/com/epam/drill/logging/LoggingConfiguration.kt` -> Just link to that file in README.md _and_ DEVELOPMENT.md